src/index.js

```diff
@@ -85,7 +85,22 @@ const Amplitude = {
     }
 
     amplitude.getInstance().setUserId(userID);
+  },
+
+  /**
+   * setUserProperties:
+   *
+   * @userProps userProps {Object} required
+   */
+  setUserProperties: function (userProps) {
+    console.log('amplitude userProps...');
```

Comment:
Can you wrap this in `_debug` and use `log()` instead? Look at some of the examples above in method: `amplitude`

Reply:
or remove it?